rating - snackbar/snackbar/snackbar.css

```diff
@@ -0,0 +1,14 @@
+@import url(/style.css);
+
+.snack {
+    position: fixed;
+    color: var(--secondary-bg);
+    background-color: #44403c;
+    padding: 0.8rem;
```

Comment:
you can move hex code to where --secondary-bg is defined.

Reply:
yeah will do it, since it is a one time usage i did not declare it that way